### PR TITLE
Added New Term Entry HTML: Tags: `<q>` tag

### PR DESCRIPTION
**Pull request by Antonio-Riccelli**
### Description

I have added an entry for the `<q>` tag to the HTML section.
This is in the "content > html > tags" folder.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.

### CODE REVIEW

File: content/html/concepts/tags/terms/q/q.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  - 'Web Design'
+Tags:
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<q>` tag is used to represent a brief inline quotation.
+It accepts a `cite` attribute which will link to a source, providing further context for the quotation.
```
Comment (KTom101) [inline content/html/concepts/tags/terms/q/q.md:15]:
```suggestion
It accepts a `cite` attribute which will link to a source that provides further context for the quotation.
```

File: content/html/concepts/tags/terms/q/q.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+## Example
+
+```html
+<!-- The text between the q tags will be surrounded by quotation marks -->
+<p>
+  The World Wide Web Consortium's idea of the Web includes
+  <q cite="https://www.w3.org/Consortium/mission"
+    >participation, sharing knowledge, and thereby building trust on a global
+    scale.</q
```
Comment (KTom101) [inline content/html/concepts/tags/terms/q/q.md:35]:
```suggestion
    scale.</q>
```

File: content/html/concepts/tags/terms/q/q.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  >
```
Comment (KTom101) [inline content/html/concepts/tags/terms/q/q.md:36]:
```suggestion
```

File: content/html/concepts/tags/terms/q/q.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+Its `cite` attribute should be a valid URL.
+
+## Example
+
+```html
+<!-- The text between the q tags will be surrounded by quotation marks -->
+<p>
+  The World Wide Web Consortium's idea of the Web includes
+  <q cite="https://www.w3.org/Consortium/mission"
```
Comment (KTom101) [inline content/html/concepts/tags/terms/q/q.md:33]:
```suggestion
  <q cite="https://www.w3.org/Consortium/mission">
```

File: content/html/concepts/tags/terms/q/q.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    >participation, sharing knowledge, and thereby building trust on a global
```
Comment (KTom101) [inline content/html/concepts/tags/terms/q/q.md:34]:
```suggestion
    participation, sharing knowledge, and thereby building trust on a global
```

File: content/html/concepts/tags/terms/q/q.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+The `<q>` tag is used to represent a brief inline quotation.
+It accepts a `cite` attribute which will link to a source, providing further context for the quotation.
+
+## Syntax
+
+```html
+<q><!-- brief quotation goes inside the tag --></q>
+```
+
+The `<q>` tag requires an opening and closing tag wrapped around a line of text. By default, the browser will render this enclosed in quotation marks.
```
Comment (KTom101) [inline content/html/concepts/tags/terms/q/q.md:23]:
```suggestion
The `<q>` tag requires an opening and closing tag wrapped around a line of text. The browser will automatically enclose the text in quotation marks.
```

File: content/html/concepts/tags/terms/q/q.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+Its `cite` attribute should be a valid URL.
```
Comment (KTom101) [inline content/html/concepts/tags/terms/q/q.md:25]:
```suggestion
The `cite` attribute should be a valid URL.
```

**Review by KTom101 — COMMENTED** [review]
@Antonio-Riccelli Just finished the 1st review for this entry. Please let us know when you're done making changes. Thanks!

**Comment by Antonio-Riccelli** [comment]
Hi @KTom101, 
I've implemented the suggested corrections, but now the request doesn't pass the "format:verify" test. What should I do? 
Thank you

**Comment by SSwiniarski** [comment]
@Antonio-Riccelli, failing that test means there's some formatting issue it doesn't like. (Usually an extra space or linefeed, which can be difficult to spot.) If you run "yarn format" on the file, and commit the changes, it will tend to fix these issues.

@KTom101, once you approve the changes, I can go ahead with review 2.

**Comment by Dusch4593** [comment]
> @Antonio-Riccelli, failing that test means there's some formatting issue it doesn't like. (Usually an extra space or linefeed, which can be difficult to spot.) If you run "yarn format" on the file, and commit the changes, it will tend to fix these issues.

If you make changes locally, on a branch of your forked repo, there's a script that should run whenever you `git add` and `git commit` your changes.

It's usually pretty good about catching those spaces and linefeeds (though I noticed the other day that Prettier doesn't seem to work with trailing spaces of code embedded in snippets or the markdown itself).

**Review by KTom101 — APPROVED** [review]
@SSwiniarski You can move on with review 2.

**Comment by SSwiniarski** [comment]
@Antonio-Riccelli, I still see several of @KTom101's suggestions unresolved. Can you address them so I can do review two?

**Comment by Antonio-Riccelli** [comment]
Hi @SSwiniarski - working on it, I should push the corrections shortly.

File: content/html/concepts/tags/terms/q/q.md (hunk 1 above)
Comment (Antonio-Riccelli) [reply content/html/concepts/tags/terms/q/q.md]: Corrected.

File: content/html/concepts/tags/terms/q/q.md (hunk 6 above)
Comment (Antonio-Riccelli) [reply content/html/concepts/tags/terms/q/q.md]: Corrected.

**Comment by Dusch4593** [comment]
@Antonio-Riccelli Hmmm let me have a look real quick and see. 😄

**Comment by Dusch4593** [comment]
@Antonio-Riccelli Try adding a space between the text and the angle-brackets.

~~`<q cite="https://www.w3.org/Consortium/mission">participation ...`~~ -> `<q cite="https://www.w3.org/Consortium/mission"> participation ...`

Then run `yarn format` again. It should look better, but let me know what you think. 😄

**Comment by Antonio-Riccelli** [comment]
@Dusch4593 Thank you. It doesn't break the tag now, but it's failing the test...

**Comment by Dusch4593** [comment]
Did you re-add/commit/push the format from your local branch?

**Comment by Antonio-Riccelli** [comment]
Yes, I ran the script, then re-added, committed and pushed.  
Should I have done it differently?

**Comment by Antonio-Riccelli** [comment]
Hi @Dusch4593, 
I've tried again. Let me know if I'm doing it correctly, please.

**Comment by Dusch4593** [comment]
@Antonio-Riccelli looks like it did! Nice work! (btw those recorded GIFs are super helpful, thank you! 😃)

File: content/html/concepts/tags/terms/q/q.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<q>` tag is used to represent a brief inline quotation.
+It accepts a `cite` attribute which will link to a source that provides further context for the quotation.
+
+## Syntax
+
```
Comment (Dusch4593) [inline content/html/concepts/tags/terms/q/q.md:19]:
```suggestion

The `<q>` tag requires an opening and closing tag wrapped around a line of text. The browser will automatically enclose the text in quotation marks:

```

File: content/html/concepts/tags/terms/q/q.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+```html
+<q><!-- brief quotation goes inside the tag --></q>
```
Comment (Dusch4593) [inline content/html/concepts/tags/terms/q/q.md:21]:
```suggestion
<q cite=""><!-- brief quotation goes inside the tag --></q>
```

File: content/html/concepts/tags/terms/q/q.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+```
+
+The `<q>` tag requires an opening and closing tag wrapped around a line of text. The browser will automatically enclose the text in quotation marks.
+
+The `cite` attribute should be a valid URL.
```
Comment (Dusch4593) [inline content/html/concepts/tags/terms/q/q.md:26]:
```suggestion
It accepts a `cite` attribute that links to a valid URL that provides further context for the quotation.
```

File: content/html/concepts/tags/terms/q/q.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+Tags:
+  - 'Tags'
+  - 'Attributes'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<q>` tag is used to represent a brief inline quotation.
+It accepts a `cite` attribute which will link to a source that provides further context for the quotation.
```
Comment (Dusch4593) [inline content/html/concepts/tags/terms/q/q.md:16]:
```suggestion
The `<q>` tag is used to represent a brief inline quotation, as opposed the [`<blockquote>` tag](https://www.codecademy.com/resources/docs/html/tags/blockquote) being used for longer ones.

```

File: content/html/concepts/tags/terms/q/q.md [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+<!-- The text between the q tags will be surrounded by quotation marks -->
+<p>
+  The World Wide Web Consortium's idea of the Web includes
+  <q cite="https://www.w3.org/Consortium/mission">
+    participation, sharing knowledge, and thereby building trust on a global
+    scale.
+  </q>
+  One can't help but agree with it!
+</p>
+```
```
Comment (Dusch4593) [inline content/html/concepts/tags/terms/q/q.md:40]: Could we include an image of what the rendered text would look like on a browser? (preferably .jpg or .png)

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
Apologies for the delay, @Antonio-Riccelli. I just left some comments/suggestion for you.

**Comment by Antonio-Riccelli** [comment]
Thank you, I'll address these today. @Dusch4593

File: content/html/concepts/tags/terms/q/q.md (hunk 12 above)
Comment (Antonio-Riccelli) [reply content/html/concepts/tags/terms/q/q.md:40]: Hi @Dusch4593, where I should store the image?

Comment (Dusch4593) [reply content/html/concepts/tags/terms/q/q.md:40]: You can save it either as a `.jpg` or `.png` file and save it in the folder for your PR branch (`terms/q/`). And then you add it with the usual  markdown image syntax.

Comment (Antonio-Riccelli) [reply content/html/concepts/tags/terms/q/q.md:40]: Thank you! Addressing these changes today.

**Comment by Antonio-Riccelli** [comment]
Hi @Dusch4593, 
Please let me know if I've implemented suggestions correctly. 
I think that test "Push/ci (test)" is failing because of the .jpg image I've added. Is that in the right folder?

**Comment by Dusch4593** [comment]
> Hi @Dusch4593, Please let me know if I've implemented suggestions correctly. I think that test "Push/ci (test)" is failing because of the .jpg image I've added. Is that in the right folder?

Hey @Antonio-Riccelli! 

First of all, thank you for uploading this awesome image! It looks great! 👍 

Secondly, I apologize for misinforming you about where to put this image! 🤭 Could we actually move it to [the `media/` folder in the root directory of this project](https://github.com/Codecademy/docs/tree/main/media)? If you're not comfortable doing this, however, I'm more than happy to make the change. 😃

Afterwards, that one check should pass.

File: content/html/concepts/tags/terms/q/q.md [hunk 13]
```diff
@@ -0,0 +1,39 @@
...
+  - 'Web Design'
+Tags:
+  - 'Tags'
+  - 'Attributes'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<q>` tag is used to represent a brief inline quotation, as opposed the [`<blockquote>` tag](https://www.codecademy.com/resources/docs/html/tags/blockquote) being used for longer ones.
```
Comment (yangc95) [inline content/html/concepts/tags/terms/q/q.md:15]:
```suggestion
The `<q>` tag is used to represent a brief inline quotation, as opposed to the [`<blockquote>` tag](https://www.codecademy.com/resources/docs/html/tags/blockquote) being used for longer ones.
```

File: content/html/concepts/tags/terms/q/q.md [hunk 14]
```diff
@@ -0,0 +1,39 @@
...
+
+## Syntax
+
+```html
+<q cite=""><!-- brief quotation goes inside the tag --></q>
```
Comment (yangc95) [inline content/html/concepts/tags/terms/q/q.md:20]: For these comments, I'm thinking that we should capitalize the first word and end with a punctuation~

**Review by yangc95 — COMMENTED** [review]
@Antonio-Riccelli while you're working on moving the image, I thought I would include another review for this entry.

Thanks again for your contribution 🙏 Please tag me or @Dusch4593 after you've resolved all the comments/suggestions ☺️

File: content/html/concepts/tags/terms/q/q.md (hunk 14 above)
Comment (yangc95) [reply content/html/concepts/tags/terms/q/q.md:20]:
```suggestion
<q cite=""><!-- Brief quotation goes inside the tag. --></q>
```

**Comment by Antonio-Riccelli** [comment]
Hi @yangc95,
I've just implemented the changes and moved the image. It seems to have worked. Let me know if further amendments are required :)

**Review by Dusch4593 — APPROVED** [review]
@Antonio-Riccelli Things LGTM on my end! 👍 There was one more suggested change I had left outstanding if you wanted to check it out and resolve it?

File: content/html/concepts/tags/terms/q/q.md (hunk 8 above)
Comment (Antonio-Riccelli) [reply content/html/concepts/tags/terms/q/q.md:18]: Hey @Dusch4593 - apologies for missing this one. Before I commit and to ensure I understand correctly, should the final result look like this: 

image

Comment (Dusch4593) [reply content/html/concepts/tags/terms/q/q.md:18]: Yep, that's right! 👍

**Comment by Dusch4593** [comment]
Thanks @Antonio-Riccelli! @yangc95 anything left to check before we approve/merge?

**Review by yangc95 — APPROVED** [review]
LGTM!

**Comment by Dusch4593** [comment]
Congrats @Antonio-Riccelli your contribution is now live on Docs! 

https://www.codecademy.com/resources/docs/html/tags/q

**Comment by Dusch4593** [comment]
@Antonio-Riccelli Hmm looks like that image didn't render the way we expected:

Screen Shot 2022-03-04 at 12 58 23 PM

I'll work on a fix right now! 👍

**Comment by Antonio-Riccelli** [comment]
@Dusch4593 Let me know if I have to change anything 😯 It seemed to render fine locally.

**Comment by Dusch4593** [comment]
We just merged [a patch PR](https://github.com/Codecademy/docs/pull/591) to hopefully fix the problem. Let us know if it's still rendering ok on your side. 😄

**Pull request merged**